Clinical-Genomics/scout
# Move variant evaluation terms from constants to database

This PR:
- Moves variant evaluation terms from constants to database. All server functions will interact with the database to retrieve and use these terms. Terms moved to database: `dismissal_term`, `manual_rank`, `cancer_tier`, `mosaicism_options`.
- Introduces the possibility to **replace the default evaluation terms with a list of custom terms** from a json file

**Previous default terms won't change and all variants previously tagged with those terms will continue showing these evaluations.**

#**Default evaluation terms**
**How to test -> Server that has already tagged variants, local demo or clinical-db stage**:
1. On master branch, tag rare and cancer cases variants (manual rank, dismiss, cancer tier, mosaicism)
2. Switch to this branch and make sure that the variants page of all types of variants is displaying a warning because these terms are missing from the database. Previously evaluated variants won't show any evaluation
3. From the command line, load default evaluation terms with the command: 
```
scout --demo load evaluation-terms default
```
4. Evaluation terms should reappear on the evaluated variants now

#**Default evaluation terms**
**How to test -> New server setup**:
1. Run `scout setup demo` and use the evaluation tags on all types of variants from rare and cancer cases. You shouldn't see any difference from master on the pages and in how the variants are tagged in the database. 

#**Custom evaluation terms**
**How to test -> Scout local instance**:
1. Replace default terms with custom terms contained in a json file. Command: 
```
scout --demo load evaluation-terms custom --file path_2_/scout/resources/variant_evaluation_terms.json
```
Since this file contains the same terms as the default terms, after loading the terms you shouldn't see any difference using these tags in the app.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
